src/frontends/pytorch/src/op_table.cpp

```diff
@@ -482,6 +483,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::masked_scatter_", op::inplace_op<op::translate_masked_scatter>},
         {"aten::matmul", op::translate_1to1_match_2_inputs<opset10::MatMul>},
         {"aten::max", op::translate_max},
+        {"aten::mv", op::translate_mv},
```

Comment:
Please use a generic helper for this operation like here:
https://github.com/openvinotoolkit/openvino/blob/2a34b745aefd787b7df3fc8ec09edfdf2e6390fd/src/frontends/pytorch/src/op_table.cpp#L499
Seems like it should be enough in your case.

Reply:
Sorry sir for late reply, I was traveling yesterday so could not reply. I will try my best to solve it today

Reply:
// Ensure matrix input is in float32 dtype
 auto matrix = context.mark_node(std::make_shared<v0::Convert>(context.get_input(0), element::f32));
// Ensure vector input is in float32 dtype
auto vector = context.mark_node(std::make_shared<v0::Convert>(context.get_input(1), element::f32));   

replace with this                                                                                                                                                                         
auto matrix = context.get_input(0);
auto vector = context.get_input(1);